core/lib/resources/font_manager.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class FontManager {
+  static const String poppins = 'Poppins';
+}
+
+enum FontWeightEnum { thin, bold, black, medium }
+
+extension FontWeightEx on FontWeightEnum {
```

Comment:
Replace it! You should create const fonts styles and use it. if you need change color you can use copyWith .
`class AppFonts {
  static TextStyle normal13 = TextStyle(
    fontFamily: GoogleFonts.roboto().fontFamily,
    fontWeight: FontWeight.w400,
    fontSize: 13,
  );`

`Text(
                        '100.0',
                        style: AppFonts.bold16.copyWith(
                          color: colors.matterhorn,
                        ),
                      ),`